erp/schema.py

```diff
@@ -39,10 +39,12 @@
 EQUIPEMENT_MALENTENDANT_BIM = "bim"
 EQUIPEMENT_MALENTENDANT_LSF = "lsf"
 EQUIPEMENT_MALENTENDANT_SCD = "scd"
+EQUIPEMENT_MALENTENDANT_LPC = "lpc"
 EQUIPEMENT_MALENTENDANT_CHOICES = [
     (EQUIPEMENT_MALENTENDANT_BIM, "Boucle à induction magnétique"),
     (EQUIPEMENT_MALENTENDANT_LSF, "Langue des signes française"),
     (EQUIPEMENT_MALENTENDANT_SCD, "Service de communication à distance"),
+    (EQUIPEMENT_MALENTENDANT_LPC, "Langue Française Parlée Complétée (LFPC)"),
```

Comment:
Maybe order this one just after LSF, for consistency

Reply:
Sure